javascript/src_js/index.d.ts

```diff
@@ -7,9 +7,9 @@
  * @format
  */
 
-import type {Yoga} from './wrapAsm';
+import type { Yoga } from "./wrapAsm";
```

Comment:
I forgot that I had told ESLint to ignore `.d.ts` files. https://github.com/facebook/yoga/blob/c09405d58c624fd26d5e0a7ed98bd10c5cc4c05b/javascript/.eslintrc.js#L12

I did that at the time because I didn't want to spend the time then to set up `typescript-eslint` a TS dep, and tests against the types. But it would be great to fix that up eventually.